projects/plugins/jetpack/modules/protect/blocked-login-page.php

```diff
@@ -608,10 +725,12 @@ function display_page( $title, $message, $back_button = false, $recovery_form =
 					<path fill="#86A6BD" d="M48.1,121.4l2.9-6.2c0.3-0.6,0.2-1.3-0.3-1.8c-1-1-1.5-2.5-1.2-4c0.3-1.7,1.7-3.1,3.4-3.4 c2.9-0.6,5.4,1.6,5.4,4.4c0,1.2-0.5,2.3-1.3,3.1c-0.5,0.5-0.6,1.2-0.3,1.8l2.9,6.2c0.1,0.2-0.1,0.5-0.3,0.5H48.4 C48.1,121.9,48,121.6,48.1,121.4"/>
 				</svg>
 
-				<?php echo $message; ?>
-				<?php if ( $recovery_form ) {
-					echo $this->get_html_recovery_form();
-				} ?>
+				<?php echo $message; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
```

Comment:
May be worth adding a comment that message includes some HTML, and we set the markup ourselves?